from django.core.exceptions import ValidationError


def validate_humidity(value: int) -> None:
    pass


def validate_temperature(value: int) -> None:
    pass


def validate_wind(value: int) -> None:
    pass


def validate_condition(value: int) -> None:
    pass
